indico/htdocs/js/indico/modules/designer.js

```diff
@@ -288,148 +307,169 @@
 
     // This function displays all the items in the 'items' array on the screen
     // If there are already some items being displayed, it does not erase them
-    function displayItems() {
-        $.each(items, function(i, item) {
-            var newDiv = createDiv(item);
+    function displayItems(isBackside) {
+        var itemsList = isBackside ? backsideItems : items;
+        $.each(itemsList, function(i, item) {
+            var newDiv = createDiv(item, isBackside);
             newDiv.css({
                 left: zoom(item.x) + 'px',
                 top: zoom(item.y) + 'px'
             });
             newDiv.append(item.toHTML());
-            if (item.selected) {
-                markSelected(newDiv.find('.designer-item'));
+            if (item.selected && !isBackside) {
+                selectItem(newDiv.find('.designer-item'));
+            }
+        });
+    }
+
+    function updatePreset(templateWidth, templateHeight) {
+        $('.js-preset-tool option').each(function() {
+            var $this = $(this);
+
+            if ($this.val() === 'custom') {
+                return;
+            }
+
+            var width = $this.data('width').toString();
+            var height = $this.data('height').toString();
+
+            if (width === templateWidth && height === templateHeight) {
+                $this.prop('selected', true);
+                $('.js-template-dimansion').attr('disabled', true);
             }
         });
     }
 
+    function changeTemplateSize(template, backsideTemplate) {
+        var tpl = $('.template-container');
+        var templateWidth = $('.template-width').val();
```

Comment:
However, I'd rather use `parseFloat`. I don't rly like the `+`, it's a bit of a weird way to convert a string to a number.